static/sass/_search.scss

```diff
@@ -0,0 +1,171 @@
+.search-result {
+  border-bottom: 1px solid #cdcdcd;
+  margin-bottom: 1.5em;
+}
+
+.result-url {
+  color: #888;
+  display: block;
+}
+
+.pagination {
+  &__forward {
+    float: right;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__back {
+    float: left;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    float: left;
+    padding: 0 0.5rem;
+
+    &:last-child {
+      padding-right: 0;
+    }
+
+    &:first-child {
+      padding-left: 0;
+    }
+  }
+}
+
+.search {
+  height: 48px;
+  margin-bottom: 1.5rem;
+
+  &__label {
+    display: none;
+  }
+
+  &__field, input.search__field[type='search'] {
+    margin: 0;
+    float: left;
+    box-shadow: none;
+    padding-right: 48px;
+  }
+
+  &__submit {
+    padding: 8px;
+    line-height: 0;
+    margin: 0;
+    border: 0;
+    margin: 1px 1px 1px -47px;
+    float: left;
+    width: auto;
+  }
+
+  &__image {
+    height: 30px;
+  }
+}
+
+.header-search {
+  display: none;
+  overflow: hidden;
+  padding: 6px;
+  width: 100%;
+  background-color: #ebebeb;
+
+  @media (min-width: $breakpoint-search) {
+    border-top: 1px solid darken($color-brand, 6%);
+    background-color: transparent;
+  }
+
+  .no-js & {
+    display: block;
+  }
+
+  @media (min-width: 982px) {
```

Comment:
Does this have a logical abstraction we could make into a variable? Something like "search threshold" or whatever its purpose is.